### DISCUSSION TITLE
How to select multiple nodes, for grouping

### DISCUSSION THREAD

**Discussion by Seanmclem** [tldraw/tldraw-v1#569] [Q&A] [answered]
I see there is a group feature, and a select all feature, but how can I manually select specific nodes in order to group them together? On Mac, cmd+click does nothing

**Comment by steveruizok** [comment, marked as answer]
> I see there is a group feature, and a select all feature, but how can I manually select specific nodes in order to group them together? On Mac, cmd+click does nothing

Shift click! Or drag a selection box around the shapes that you want to select.
